### PR TITLE
Wrappers: Add BossBars wrapper

### PR DESCRIPTION
**Pull request by camnwalter**
I can also add other ways to get the boss bar, but the name seems like the most reasonable one.

Let me know if you think there should be other methods added as well

### CODE REVIEW

File: src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/BossBars.kt [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+            UTextComponent(name),
+            percent,
+            color.toMC(),
+            style.toMC(),
+            shouldDarkenSky,
+            false,
+            shouldThickenFog
+        )
+
+        toMC().asMixin<BossBarHudAccessor>().bossBars[UUID.randomUUID()] = bossBar
```
Comment (mattco98) [inline src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/BossBars.kt:74]: Does it not matter that this UUID is different from the one above?

Comment (camnwalter) [reply src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/BossBars.kt:74]: Pretty sure it doesn't, but just to be sure I'll make them use the same one.

**Comment by camnwalter** [comment]
I guess I can pass in the dragonMusic param as well, but it is useless if you're not in the end

**Comment by mattco98** [comment]
Seems like it'd also be good to provide a way to remove a boss bar given the `BossBar` object, which I assume would just entail the `BossBar` holding onto its random UUID. Other than that, LGTM

**Pull request merged**